src/cmd.ts

```diff
@@ -297,9 +297,13 @@ program
 
 program
 	.command("clear-cache")
-	.description("Clear the cache of downloaded-and-rejected torrents")
+	.description(
+		"Clear the cache without causing torrents to be re-snatched and reset the timestamps for excludeOlder and excludeRecentSearch",
+	)
 	.action(async () => {
+		console.log("Clearing cache, this may take a few minutes...");
 		await db("decision").whereNull("info_hash").del();
+		await db("timestamp").del();
```

Comment:
I would not expect this to take a few minutes.

Reply:
It previously did when it was updating everything individually. But yeah it takes a few seconds now.